diff --git a/node_modules/ky/index.d.ts b/node_modules/ky/index.d.ts
index caf3335..ab46704 100644
--- a/node_modules/ky/index.d.ts
+++ b/node_modules/ky/index.d.ts
@@ -1,5 +1,3 @@
-/// <reference lib="dom"/>
-
 type Except<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;

 export type BeforeRequestHook = (options: NormalizedOptions) => void | Promise<void>;
@@ -55,7 +53,7 @@ interface KyOptions extends RequestInit {
 	Prepends the input URL with the specified prefix.
 	The prefix can be any valid URL, either relative or absolute.
 	*/
-	prefixUrl?: URL | string;
+	prefixUrl?: string;

 	/**
 	Numer of times to retry failed requests.
@@ -119,7 +117,7 @@ interface OptionsWithBody extends KyOptions {

 export type Options = OptionsWithoutBody | OptionsWithBody;

-export type Input = Request | URL | string;
+export type Input = Request | string;

 /**
 Returns a `Response` object with `Body` methods added for convenience.
